Documentation/direct-tools.md

```diff
@@ -11,12 +11,25 @@ use block and file storage directly from a pod without any of the Kubernetes mag
 although it is not meant to be used in production. All of the benefits of Kubernetes storage including failover, detach, and attach will not be available.
 If your pod dies, your mount will die with it.
 
+## Start the Direct Mount pod
+
+To test mounting your Ceph volumes, start a pod with the necessary mounts. An example is provided in the examples test directory:
+
+```console
+kubectl create -f cluster/examples/kubernetes/ceph/direct-mount.yaml
+```
+
 ## Block Storage Tools
 
 After you have created a pool as described in the [Block Storage](ceph-block.md) topic, you can create a block image and mount it directly in a pod.
 This example will show how the Ceph rbd volume can be mounted in the toolbox pod.
 
-After you have started and connected to the [Rook toolbox](ceph-toolbox.md), proceed with the following commands in the toolbox.
+After the [direct Mount Pod](direct-tools.md#Start-the-Direct-Mount-pod) is started, connect to the pod
+
+```console
```

Comment:
How about also moving this to the "Start the Direct Mount pod" section?

Reply:
Ok, so you mean to say starting and connecting to the pod under the same section?

Reply:
right